modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -13,8 +18,14 @@ def submit
             raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
           end
 
+          unless bgs_claim&.dig&.[](:filed_5103_waiver_ind) == FILE_5103
+            bgs_claim[:bnft_claim_dto][:filed_5103_waiver_ind] = FILE_5103
```

Comment:
same as above.. `:filed5103_waiver_ind`

Reply:
[9bd2000](https://github.com/department-of-veterans-affairs/vets-api/pull/10867/commits/9bd2000d5d5b83bb74cb7e9fb05d35ff8286b8c9)